frame/support/procedural/src/construct_runtime/expand/config.rs

```diff
@@ -32,11 +33,11 @@ pub fn expand_outer_config(
 	for decl in pallet_decls {
 		if let Some(pallet_entry) = decl.find_part("Config") {
 			let config = format_ident!("{}Config", decl.name);
-			let mod_name = decl.pallet.mod_name();
+			let pallet_name = &decl.name.to_string().to_snake_case();
 			let field_name = if let Some(inst) = decl.instance.as_ref() {
-				format_ident!("{}_{}", mod_name, inst)
+				format_ident!("{}_{}", pallet_name, inst.to_string().to_snake_case())
```

Comment:
This here is technically a breaking change as well, unsure if it's worth?

Reply:
we don't need to concatenate with the instance name, the pallet name is already unique, and I think it worth it, the name is better than the path to identify the pallet